### DISCUSSION TITLE
how to change image link of the logo

### DISCUSSION THREAD

**Discussion by huhas12** [FreshRSS/FreshRSS#4918] [Q&A]
Hello, 

Could anyone please help me to change the logo image link? How can I do it?

**Comment by Alkarex** [comment]
Hello,
Edit the `logo_html` property of your `./data/config.php`:

https://github.com/FreshRSS/FreshRSS/blob/543fa4e76c1761154801febd08400520b75143e3/config.default.php#L3-L5

https://github.com/FreshRSS/FreshRSS/blob/543fa4e76c1761154801febd08400520b75143e3/config.default.php#L34-L37

**Reply by huhas12** [reply]
Thank you , but I mean the link behind the logo image.

**Reply by math-GH** [reply]
Why do you need to change the link?
